SecureImage/UI/Albums/AlbumsViewController.swift

```diff
@@ -19,12 +19,21 @@
 //
 
 import UIKit
+import RealmSwift
+import QuartzCore
```

Comment:
Shouldn't need this since UIKit is imported

Reply:
Checking into this. I think QuartzCore has to be imported separately but I'll do a quick test now.

Reply:
Your right. I removed the import.